@uportal/esco-content-menu-lit/src/styles/content-grid.scss

```diff
@@ -0,0 +1,286 @@
+            border-right: 6px solid transparent;
+            pointer-events: none;
+          }
+
+          select {
+            -webkit-appearance: none;
+            -moz-appearance: none;
+            padding: 5px 24px 5px 5px;
+
+            &::-ms-expand {
+              display: none;
+            }
+          }
+        }
+
+        input,
+        select {
+          border: none;
+          height: 100%;
+
+          /* prettier-ignore */
+          box-shadow:
+            0 2px 2px 0 rgba(0, 0, 0, 0.14),
+            0 3px 1px -2px rgba(0, 0, 0, 0.12),
+            0 1px 5px 0 rgba(0, 0, 0, 0.2);
+          outline: none;
+          padding: 0;
+          margin: 0;
+        }
+
+        input {
+          line-height: 32px;
+          border-top-left-radius: 48px;
+          border-bottom-left-radius: 48px;
+          width: 60%;
+          text-indent: 10px;
+          font-size: 16px;
+        }
+
+        select {
+          border-top-right-radius: 48px;
+          border-bottom-right-radius: 48px;
+          width: 40%;
+
+          option.default {
+            font-style: italic;
+            color: grey;
+          }
+        }
+
+        > div {
+          display: none;
+        }
+      }
+    }
+
+    .preamble {
+      display: flex;
+      flex-flow: row wrap;
+      padding: 0 20px;
+    }
+
+    .flex-grid {
+      display: flex;
+      flex-flow: row wrap;
+      justify-content: var(--content-grid-flex-grid-justify, center);
+
+      .flex-item {
+        margin: 20px auto;
+        margin: var(--content-griditem-margin, 20px auto);
+        padding: 0 2.5px;
+      }
+
+      /** fix IE flexbox bug on margin auto https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/14593426/. */
+      @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
+        .flex-item {
+          margin: 20px 5px;
+        }
+      }
+    }
+
+    a.no-style {
+      text-decoration: none;
+      color: inherit;
+    }
+  }
+
+  &.small,
+  &.smaller {
+    padding: 0.5em;
+    > div {
+      border-radius: 5px;
+      background-color: #f3f3f3;
+
+      > .title {
+        position: relative;
+        border-radius: 5px;
+        background-color: #fff;
+
+        > h1,
+        > slot > h1 {
+          margin-right: $searchSize;
+        }
+
+        > .filter,
+        slot > .filter {
+          margin: auto 0;
+          position: absolute;
+          display: flex;
+          flex-flow: row nowrap;
+          width: calc(100% - 30px);
+          height: calc(100% - 10px);
+          padding: 5px 0;
+          max-width: unset;
+          right: 15px;
+
+          &.closed {
+            width: $searchSize;
+
+            > input,
+            > select,
+            &::after {
+              visibility: hidden;
+              padding: 0;
+              display: none;
+            }
+          }
+
+          > div {
+            display: -webkit-box;
```

Comment:
Would it make sense to leverage a tool like https://github.com/postcss/autoprefixer#readme
to automate this and keep it in sync with the supported browserslist?

Reply:
Add it in [0f3ca73](https://github.com/uPortal-Project/uPortal-web-components/pull/919/commits/0f3ca73f61a9f505039d665dda862cebc9502fc8)